core/src/epicli/cli/helpers/input_query.py

```diff
@@ -0,0 +1,5 @@
+def prompt_for_value(prompt):
+    input_text = ''
+    while input_text == '':
+        input_text = input(prompt)
+    return input_text
```

Comment:
If you want use this also for passwords it would be okay to use getpass or equivalent.